api/src/controllers/request.ts

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { errorWithCode, logger } from '@bcgov/common-nodejs-utils';
+import { Response } from 'express';
+import DataManager from '../db';
+import { HumanActionType, RequestType } from '../db/model/request';
+import { AuthenticatedUser } from '../libs/authmware';
+import { fulfillRequest } from '../libs/fulfillment';
+import { archiveProjectSet } from '../libs/profile';
+import shared from '../libs/shared';
+
+const dm = new DataManager(shared.pgPool);
+
+export const fetchHumanActionRequests = async (
+  { query }: { query: any }, res: Response
+): Promise<void> => {
+  const { RequestModel } = dm;
+  const { filter } = query;
+
+  try {
+    // Step 1. fetch all requests with matching filter
+    const requests = await RequestModel.findActiveByFilter(filter, true);
```

Comment:
In the code above you do `const requiresHumanAction = true;` I'd follow that convention for this true to make the code more readable. As I read it, not sure what impact `true` has here.